charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -226,6 +226,7 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `controller.adminUser`                | Admin username (and password) created as a secret if adminSecret is true | `admin` |
 | `controller.adminPassword`            | Admin password (and user) created as a secret if adminSecret is true | Random value |
 | `controller.additionalSecrets`        | List of additional secrets to create and mount according to [JCasC docs](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/secrets.adoc#kubernetes-secrets) | `[]` |
+| `controller.additionalExistingSecrets`| List of additional existing secrets to create and mount according to [JCasC docs](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/secrets.adoc#kubernetes-secrets) | `[]` |
```

Comment:
```suggestion
| `controller.additionalExistingSecrets`| List of additional existing secrets to mount according to [JCasC docs](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/secrets.adoc#kubernetes-secrets) | `[]` |
```
Those secrets need to already exist. That's also what `controller.additionalExistingSecrets` indicates.

Reply:
thanks, I've copied the row from `additionalSecrets` and missed the create part, updated